zenoh/src/net/routing/mod.rs

```diff
@@ -168,4 +169,18 @@ impl RoutingContext<NetworkMessage> {
         }
         None
     }
+
+    #[inline]
+    pub(crate) fn full_key_expr(&self) -> Option<OwnedKeyExpr> {
```

Comment:
I am not sure that it is most optimal way to generate KeyExpr.
Maybe during the message processing we have better time to create and cache it?

Reply:
It could be a future optimization yes.